.terraformignore

```diff
@@ -0,0 +1,2 @@
+examples
+examples/*
```

Comment:
Is this just for the `terraform validate` in pre-commit?  If the above comment works, then drop this file.